deploy/examples/create-external-cluster-resources.py

```diff
@@ -559,16 +559,19 @@ def endpoint_dial(self, endpoint_str, timeout=3, cert=None):
         for prefix in protocols:
             try:
                 ep = "{}://{}".format(prefix, endpoint_str)
+                verify = None
                 # If verify is set to a path to a directory,
                 # the directory must have been processed using the c_rehash utility supplied with OpenSSL.
                 if prefix == "https" and cert and self._arg_parser.rgw_skip_tls:
+                    verify = False
                     r = requests.head(ep, timeout=timeout, verify=False)
                 elif prefix == "https" and cert:
+                    verify = cert
                     r = requests.head(ep, timeout=timeout, verify=cert)
                 else:
                     r = requests.head(ep, timeout=timeout)
                 if r.status_code == 200:
-                    return prefix
+                    return prefix, verify
             except:
                 continue
         raise ExecutionFailureException(
```

Comment:
It will be better if the error message is more understanding, say if the SSL cert is wrong then mention that it is SSL failure, or the endpoint won't be accessible. More clarity to error message is good IMO

Reply:
How to get which type of error is this??

From the respone??